### PR TITLE
[BC-18080] Add Binaries methodologies

### PR DESCRIPTION
**Pull request by bharat619**
Jira: [BC-18080](https://bugcrowd.atlassian.net/browse/BC-18080l)

Methodology Document: [Binary Methodology](https://docs.google.com/document/d/1gz2YLOMkGhI9huhJpdDMOtk73JZjHbxADMKtzL7ClhI/edit#)

To Test:
Content must be same as in the methodology document

### CODE REVIEW

File: methodologies/binaries.json [hunk 1]
```diff
@@ -0,0 +1,240 @@
+{
+  "metadata": {
+    "title": "Binaries",
+    "release_date": "2022-01-10T00:00:00+00:00",
+    "description": "Bugcrowd Binaries testing methodology",
```
Comment (arcwhite) [inline methodologies/binaries.json:5]: I think stick with "Bugcrowd Binary testing methodology" (e.g. singular binary, not binaries)

File: methodologies/binaries.json [hunk 2]
```diff
@@ -0,0 +1,240 @@
+{
+  "metadata": {
+    "title": "Binaries",
```
Comment (arcwhite) [inline methodologies/binaries.json:3]: Likewise here, Binary over binaries I think?

File: methodologies/binaries.json [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+    "release_date": "2022-01-10T00:00:00+00:00",
+    "description": "Bugcrowd Binaries testing methodology",
+    "vrt_version": "10.0.1"
+  },
+  "content": {
+    "steps": [
+      {
+        "key": "insufficient_authentication_authorization",
+        "title": "B1 Insufficient Authentication/Authorization",
```
Comment (arcwhite) [inline methodologies/binaries.json:12]: Is putting the number in the title correct? I think we want this to just be a simple title string, no decoration

File: methodologies/binaries.json (hunk 1 above)
Comment (bharat619) [reply methodologies/binaries.json]: Sure

File: methodologies/binaries.json (hunk 2 above)
Comment (bharat619) [reply methodologies/binaries.json]: Chaged

File: methodologies/binaries.json (hunk 3 above)
Comment (bharat619) [reply methodologies/binaries.json]: The doc had the B1 B2 etc title in the string. So had just added it. 
I have removed it

**Review by arcwhite — APPROVED** [review]

**Review by arcwhite — APPROVED** [review]

**Pull request merged**